### PR TITLE
Create patchouli books translations of simplified Chinese , correct book content and update translation tools

### PR DESCRIPTION
**Pull request by VisoTC**

### CODE REVIEW

File: src/main/resources/data/tfc/patchouli_books/field_guide/zh_cn.json
Comment (alcatrazEscapee) [inline src/main/resources/data/tfc/patchouli_books/field_guide/zh_cn.json]: This shouldn't be committed here - this file is unused at runtime and only exists as a single location for localization data. My preference would be to move it to `/resources/lang/zh_cn.json` and make `generate_book.py` point to the `./lang` folder automatically.

File: resources/i18n.py
Comment (alcatrazEscapee) [inline resources/i18n.py]:
```suggestion
            json.dump(self.after, f, indent=2, ensure_ascii=False)
```

**Review by alcatrazEscapee — CHANGES_REQUESTED** [review]

File: src/main/resources/data/tfc/patchouli_books/field_guide/zh_cn.json
Comment (VisoTC) [reply src/main/resources/data/tfc/patchouli_books/field_guide/zh_cn.json]: This is nice so that the translation files can be used for git without adding extra files

File: src/main/resources/data/tfc/patchouli_books/field_guide/zh_cn/categories/mechanics.json
Comment (gaogao-qwq) [inline src/main/resources/data/tfc/patchouli_books/field_guide/zh_cn/categories/mechanics.json:4]: 鉴于群峦传说的红钢和蓝钢的独特性，也许这里用红蓝钢代替彩钢会更好？

File: src/main/resources/data/tfc/patchouli_books/field_guide/zh_cn/entries/getting_started/a_place_to_sleep.json
Comment (gaogao-qwq) [inline src/main/resources/data/tfc/patchouli_books/field_guide/zh_cn/entries/getting_started/a_place_to_sleep.json]: 专有名词前后不一致

File: src/main/resources/data/tfc/patchouli_books/field_guide/zh_cn/entries/getting_started/a_place_to_sleep.json
Comment (gaogao-qwq) [inline src/main/resources/data/tfc/patchouli_books/field_guide/zh_cn/entries/getting_started/a_place_to_sleep.json]: 专有名词前后不一致。
`然后，右键单击$(thing)大块未加工皮革$()。`
建议改为
`然后，手持$(thing)大块未加工皮革$()右击干草块。`
删除`要制作茅草床，请将两个彼此相邻的$(thing)干草块$()放置在一起。 `句号末尾空格

File: src/main/resources/data/tfc/patchouli_books/field_guide/zh_cn/entries/getting_started/building_materials.json
Comment (gaogao-qwq) [inline src/main/resources/data/tfc/patchouli_books/field_guide/zh_cn/entries/getting_started/building_materials.json]: 删除`在游戏的早期阶段，建筑可能是一个挑战，因为许多坚固的建筑方块需要金属工具才能获得。 `句号末尾空格

File: src/main/resources/data/tfc/patchouli_books/field_guide/zh_cn/categories/mechanics.json
Comment (VisoTC) [reply src/main/resources/data/tfc/patchouli_books/field_guide/zh_cn/categories/mechanics.json:4]: 这里原文是：Advanced sections of the tech tree, from the first pickaxe, all the way to colored steel.
所以我认为`colored steel`翻译为`彩钢`会更合适

**Comment by alcatrazEscapee** [comment]
This looks like good work but I'm going to hold this until the book is in a bit more finished state, at least until the rest of the WIP pages are filled out. As there's a lot of editing and misc. changes going on in the meantime once conflicts are resolved.

**Comment by alcatrazEscapee** [comment]
@VisoTC I'd be ready to merge this if you updated it to fix conflicts, and updated the translation. Thanks!

**Comment by aquila-zyy** [comment]
@VisoTC Hi! May I ask that if you are still maintaining this PR? I would like to contribute to the zh-cn localization based on your translation here. I can help update your translation to the latest version of the game and maybe adding some finishing touch.

**Comment by eerussianguy** [comment]
this will be taken care of by #2082

**Pull request closed without merging**
